index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Twitter
+        </title>
+        <script src="https://kit.fontawesome.com/34321f5008.js" crossorigin="anonymous"></script>
+        <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" ></script>
+
+    </head>
+    <body>
+        <div class="content">
```

Comment:
I can see how many times of using div! I think you should use a suitable tag for each "content" such as a grid, header, or another!